authapi/serializers.py

```diff
@@ -105,3 +97,25 @@ def update(self, instance, validated_data):
 
         instance.save()
         return instance
+
+
+class NewUserSerializer(BaseUserSerializer):
+    password = serializers.CharField(
+        style={'input_type': 'password'}, write_only=True)
+
+    class Meta:
+        model = User
+        fields = (
+            'id', 'url', 'first_name', 'last_name', 'email', 'admin', 'teams',
+            'organizations', 'password', 'active')
```

Comment:
We can probably have the `Meta` on the base class, and just adjust the CharField on these classes. Is there a reason why you chose this way?